File: app/src/main/java/com/kkkcut/e20j/ui/fragment/WebFragment.java
package com.kkkcut.e20j.ui.fragment;

import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import butterknife.BindView;
import com.kkkcut.e20j.base.BaseFragment;
import com.kkkcut.e20j.us.R;

/* loaded from: classes.dex */
public class WebFragment extends BaseFragment {

    @BindView(R.id.webview)
    WebView webView;

    @Override // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    protected int getContentViewLayoutID() {
        return R.layout.fragment_web;
    }

    public static WebFragment newInstance() {
        return new WebFragment();
    }

    @Override // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    protected void initViewsAndEvents() {
        WebSettings settings = this.webView.getSettings();
        settings.setLayoutAlgorithm(WebSettings.LayoutAlgorithm.SINGLE_COLUMN);
        settings.setJavaScriptEnabled(true);
        settings.setBuiltInZoomControls(true);
        settings.setSupportZoom(true);
        this.webView.setWebViewClient(new WebViewClient());
        this.webView.loadUrl("file:////android_asset/HTMLPage1.html");
    }
}

File: app/src/main/java/com/kkkcut/e20j/ui/fragment/WebFragment.kt
package com.kkkcut.e20j.ui.fragment

import android.webkit.WebSettings
import android.webkit.WebView
import android.webkit.WebViewClient
import com.kkkcut.e20j.base.BaseFragment
import com.kkkcut.e20j.us.R

/* loaded from: classes.dex */
class WebFragment() : BaseFragment() {
    var webView: WebView? = null

    // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    override fun getContentViewLayoutID(): Int {
        return R.layout.fragment_web
    }

    // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    override fun initViewsAndEvents() {
        val settings: WebSettings = webView!!.getSettings()
        settings.setLayoutAlgorithm(WebSettings.LayoutAlgorithm.SINGLE_COLUMN)
        settings.setJavaScriptEnabled(true)
        settings.setBuiltInZoomControls(true)
        settings.setSupportZoom(true)
        webView!!.setWebViewClient(WebViewClient())
        webView!!.loadUrl("file:////android_asset/HTMLPage1.html")
    }

    companion object {
        fun newInstance(): WebFragment {
            return WebFragment()
        }
    }
}
